### DISCUSSION TITLE
Will the Atlantis Template Be Available for Nuxt?

### DISCUSSION THREAD

**Discussion by RadwaMosaad** [primefaces/community#2676] [PrimeVue Templates] [answered]
I recently purchased the Atlantis template from PrimeVue and am currently working on a Nuxt 3 project. I was wondering if there are any plans to make the Atlantis template available or compatible with Nuxt, especially with the upcoming updates? If so, is there any timeline for when it might be released?
Thank you for your assistance!

**Comment by tugcekucukoglu** [comment, marked as answer]
Currently, there is no plan to make Nuxt version of Atlantis. However, pages can be implemented easily for Nuxt. 

Thanks for your understanding.
